pkg/controller/cronjob/utils.go

```diff
@@ -173,7 +173,6 @@ func getRecentUnmetScheduleTimes(sj batchv1beta1.CronJob, now time.Time) ([]time
 func getJobFromTemplate(sj *batchv1beta1.CronJob, scheduledTime time.Time) (*batchv1.Job, error) {
 	// TODO: consider adding the following labels:
 	// nominal-start-time=$RFC_3339_DATE_OF_INTENDED_START -- for user convenience
```

Comment:
We may want to remove this one too

Reply:
Suggest removing this whole TODO, since we won't be adding labels to jobs